scene/gui/base_button.cpp

```diff
@@ -323,9 +323,7 @@ int BaseButton::get_button_mask() const {
 void BaseButton::set_enabled_focus_mode(FocusMode p_mode) {
 
 	enabled_focus_mode = p_mode;
-	if (!status.disabled) {
-		set_focus_mode(p_mode);
-	}
```

Comment:
Now that I think about it, shouldn't this be kept for compatibility? The warning is good to tell users that they should change their code, but the code should still work properly without change in the `3.2` branch.

Reply:
Yes, we can keep that. Does the warning message need any change?

Reply:
No I think the message is good. It tells users that they should change their code, but doesn't imply that the existing code is invalidated.